### PR TITLE
Tiny addition to frontend

### PR DESCRIPTION
**Pull request by nigef**
nice and basic

### CODE REVIEW

File: site/source/stylesheets/site.css.scss [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+}
+
 body {
   background-color: #f3f3f3;
   color: #333;
   font-family: -apple-system, BlinkMacSystemFont, "Avenir Next", "Avenir",
     "Segoe UI", "Lucida Grande", "Helvetica Neue", "Helvetica", "Fira Sans",
     "Roboto", "Noto", "Droid Sans", "Cantarell", "Oxygen", "Ubuntu",
     "Franklin Gothic Medium", "Century Gothic", "Liberation Sans", sans-serif;
+  margin: 0 auto;
```
Comment (arkon) [inline site/source/stylesheets/site.css.scss]: What's the use of this?

File: site/source/stylesheets/site.css.scss [hunk 2]
```diff
@@ -17,7 +22,38 @@ a {
...
+summary {
+  &:hover {
+    @extend .options-hover;
+  }
+}
+
+.options-hover {
+  color: white;
+  padding-left: 2px;
+  transition: padding 100ms;
```
Comment (arkon) [inline site/source/stylesheets/site.css.scss]: I generally don't like doing transitions on things that involve repaints and recomposition (at least in Blink). See [CSS Triggers](https://csstriggers.com/) for details.

File: site/source/stylesheets/site.css.scss [hunk 3]
```diff
@@ -17,7 +22,38 @@ a {
   }
 }
 
-details,
-li {
+section {
+  column-count: 3;
+  font-family: Monaco,"Courier New",monospace;
+  font-size: 14px;
```
Comment (arkon) [inline site/source/stylesheets/site.css.scss]: Please use `em` or `rem` values (below as well).

File: site/source/stylesheets/site.css.scss [hunk 4]
```diff
@@ -17,7 +22,38 @@ a {
   }
 }
 
-details,
-li {
+section {
+  column-count: 3;
```
Comment (arkon) [inline site/source/stylesheets/site.css.scss]: I'd like to see the vendor prefixes for this as well, since it became unprefixed fairly recently (for WebKit, iirc).

File: site/source/stylesheets/site.css.scss [hunk 5]
```diff
@@ -17,7 +22,38 @@ a {
...
+  font-family: Monaco,"Courier New",monospace;
```
Comment (arkon) [inline site/source/stylesheets/site.css.scss]: Add a space after commas.

File: site/source/stylesheets/site.css.scss [hunk 6]
```diff
@@ -1,10 +1,15 @@
+html {
+  display: flex;
```
Comment (arkon) [inline site/source/stylesheets/site.css.scss]: Why is this needed?

File: site/source/stylesheets/site.css.scss [hunk 7]
```diff
@@ -17,7 +22,38 @@ a {
...
+}
+
+summary {
+  &:hover {
+    @extend .options-hover;
+  }
+}
+
+.options-hover {
+  color: white;
```
Comment (arkon) [inline site/source/stylesheets/site.css.scss]: I prefer hex codes over keywords.

File: site/source/javascripts/site.js [hunk 8]
```diff
@@ -7,4 +7,5 @@ $(document).on('click', '.course', function(e) {
   elImg.src = courseSvg;
 
   $('#chart').html(elImg);
+  window.scrollTo(0, 0);
```
Comment (arkon) [inline site/source/javascripts/site.js]: You'll need to resolve merge conflicts with this file too.

File: site/source/stylesheets/site.css.scss (hunk 1 above)
Comment (nigef) [inline site/source/stylesheets/site.css.scss]: this is to center it

File: site/source/stylesheets/site.css.scss (hunk 6 above)
Comment (nigef) [inline site/source/stylesheets/site.css.scss]: to center the thingy

File: site/source/javascripts/site.js (hunk 8 above)
Comment (nigef) [inline site/source/javascripts/site.js]: hmm, i didn't see any, i'll look again

File: site/source/stylesheets/site.css.scss (hunk 2 above)
Comment (nigef) [inline site/source/stylesheets/site.css.scss]: i'm with you on that too. its nice to get a little movement as a visual queue. 
generally, do we make an exception for a tiny page, or do we forget about movement?

File: site/source/stylesheets/site.css.scss (hunk 4 above)
Comment (nigef) [inline site/source/stylesheets/site.css.scss]: oh true, i forgot to look

File: site/source/stylesheets/site.css.scss (hunk 5 above)
Comment (nigef) [inline site/source/stylesheets/site.css.scss]: will do, along with the other fixes...

File: site/source/stylesheets/site.css.scss (hunk 1 above)
Comment (arkon) [inline site/source/stylesheets/site.css.scss]: Why? Isn't it full width?

File: site/source/stylesheets/site.css.scss (hunk 2 above)
Comment (arkon) [inline site/source/stylesheets/site.css.scss]: You should use `transform`

File: site/source/stylesheets/site.css.scss (hunk 1 above)
Comment (nigef) [inline site/source/stylesheets/site.css.scss]: ya, now it's full width and centered 👍

Comment (arkon) [inline site/source/stylesheets/site.css.scss]: Ah wait, I thought it was at 100% width. I'd rather it be 100% width, with the selected chart as centred.

File: site/source/stylesheets/site.css.scss [hunk 9]
```diff
@@ -53,7 +50,8 @@ summary {
 }
 
 .options-hover {
-  color: white;
-  padding-left: 2px;
-  transition: padding 100ms;
+  color: #dcdcdc;
+  padding-left: 0.2rem;
+  transform: translateX(0.2rem);
```
Comment (arkon) [inline site/source/stylesheets/site.css.scss]: I think this jumps a bit too much. Also, I'd rather see:

```css
.course:hover,
summary:hover {
  // ...
}
```

File: site/source/stylesheets/site.css.scss [hunk 10]
```diff
@@ -23,24 +23,21 @@ a {
...
-  color: #60e859;
   background-color: #333;
-  padding: 10px;
   box-shadow: inset 0 1px 15px rgba(0, 0, 0, 0.8), 0 1px 0 rgba(255, 255, 255, 0.1), 0 -1px 0 rgba(0, 0, 0, 0.5);
+  color: #60e859;
+  font-size: 0.9rem;
+  padding: 1.6rem;
+  -webkit-column-count: 3;
+  -moz-column-count: 3;
+  column-count: 3;
```
Comment (arkon) [inline site/source/stylesheets/site.css.scss]: You'll need to add some media queries as well to handle narrower screens.

File: site/source/stylesheets/site.css.scss [hunk 11]
```diff
@@ -23,24 +23,21 @@ a {
 }
 
 section {
-  column-count: 3;
-  font-family: Monaco,"Courier New",monospace;
-  font-size: 14px;
-  color: #60e859;
   background-color: #333;
-  padding: 10px;
   box-shadow: inset 0 1px 15px rgba(0, 0, 0, 0.8), 0 1px 0 rgba(255, 255, 255, 0.1), 0 -1px 0 rgba(0, 0, 0, 0.5);
```
Comment (arkon) [inline site/source/stylesheets/site.css.scss]: You could just make use of Sass's `rgba` function and do `rgba(#000, 0.8)` if you want.

**Comment by arkon** [comment]
@nigef Do you intend on opening another PR for this?

**Comment by nigef** [comment]
I realized my additions are trivial. I was just procrastinating on something else. Probably not.

**Comment by arkon** [comment]
@nigef I added some of these changes in. Thanks anyway. 👍
